### PR TITLE
Add troubleshooting tip about ca error

### PR DESCRIPTION
**Pull request by nannanli**
Signed-off-by: nannanli <nannanli@cn.ibm.com>

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [ ] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
Please describe your pull request.

:heart:Thank you!

### CODE REVIEW

File: docs/troubleshoot/troubleshoot-apiml.md [hunk 1]
```diff
@@ -308,3 +308,29 @@ The Zowe started task needs to run under the same user ID as z/OSMF (typically I
 The hostname that is displayed in the details of the exception is a valid hostname. You can validate that the hostname is valid by using `ping` command on the same mainframe system. For example, `ping USILCA32.lvn.broadcom.net`. If it is valid, then the problem can be caused by insufficient privileges of your started task that is not allowed to do network access.
 
 You can fix it by setting up the security environment as described in the [Zowe documentation](../user-guide/configure-zos-system.html#configure-security-environment-switching).
+
+### Certificate error when using an external CA to deploy Zowe
```
Comment (std4lqi) [inline docs/troubleshoot/troubleshoot-apiml.md:312]: How about `Certificate error when using both an external certificate and Single Sign-On to deploy Zowe`?

File: docs/troubleshoot/troubleshoot-apiml.md [hunk 2]
```diff
@@ -308,3 +308,29 @@ The Zowe started task needs to run under the same user ID as z/OSMF (typically I
...
+
+**Symptom:**
+
+You used an external CA to deploy Zowe. When you log in to the Zowe Desktop, you encounter an error similar to the following:
```
Comment (std4lqi) [inline docs/troubleshoot/troubleshoot-apiml.md:316]: Mention Single Sign-On as well please. Like `You used an external certificate and Single Sign-On to deploy Zowe`

**Review by std4lqi — APPROVED** [review]
Approve with some comments. Thanks!

File: docs/troubleshoot/troubleshoot-apiml.md [hunk 3]
```diff
@@ -308,3 +308,29 @@ The Zowe started task needs to run under the same user ID as z/OSMF (typically I
...
+}
+```
+
+**Solution:**
+
+This issue might occur when you use a Zowe version of 1.12.0 or later. To resolve the issue, you can download your external root certificate and intermediate certificates in PEM format. Then, add the following parameter in the Zowe `instance.env` file.
+ 
+```ZWED_node_https_certificateAuthorities="/path/to/zowe/keystore/local_ca/localca.cer-ebcdic","/path/to/carootcert.pem","/path/to/caintermediatecert.pem"```
+ 
+Recycle your zone server. You should be able to log in to the Zowe Desktop successfully now.
```
Comment (rjmaomao) [inline docs/troubleshoot/troubleshoot-apiml.md:336]: should be "zowe" server, zone is a typo.

**Comment by nannanli** [comment]
@1000TurquoisePogs Hi Sean, could you review this PR as well? Thank you!

**Review by PlutoZhang — APPROVED** [review]

**Pull request merged**